components/pages/Vote.tsx

```diff
@@ -1,11 +1,26 @@
-import { Banner, HowItWorks, Layout, Votes } from "components";
+import { Banner, HowItWorks, Layout, LoadingSpinner, PageInnerWrapper, PageOuterWrapper, Votes } from "components";
+import { useVotesContext } from "hooks";
+import { LoadingSpinnerWrapper, Strong } from "./styles";
 
 export function Vote() {
+  const { getUserIndependentIsLoading } = useVotesContext();
   return (
     <Layout>
-      <Banner />
+      <Banner>
+        Stake, vote &amp; earn up to <Strong>30% APY</Strong>
+      </Banner>
       <HowItWorks />
-      <Votes />
+      <PageOuterWrapper>
```

Comment:
wrapper components live in the page component now, more convenient to share them this way